### PR TITLE
fix type conversion warnings 

### PR DESCRIPTION
**Pull request by ilqvya**
I hate warnings in my warnings list!

### CODE REVIEW

File: cocos/2d/CCLabel.h [hunk 1]
```diff
@@ -410,7 +410,7 @@ class CC_DLL Label : public Node, public LabelProtocol, public BlendProtocol
     /**
     * Return the outline effect size value.
     */
-    int getOutlineSize() const { return _outlineSize; }
+    int getOutlineSize() const { return static_cast<int>(_outlineSize); }
```
Comment (minggo) [inline cocos/2d/CCLabel.h]: I think we should return float instead of cast internally.
@zilongshanren

File: cocos/3d/CCTerrain.h [hunk 2]
```diff
@@ -424,7 +424,7 @@ class CC_DLL Terrain : public Node
     /**
      * get the terrain's size
      */
-    Size getTerrainSize() const { return Size(_imageWidth, _imageHeight); }
+    Size getTerrainSize() const { return Size(static_cast<float>(_imageWidth), static_cast<float>(_imageHeight)); }
```
Comment (minggo) [inline cocos/3d/CCTerrain.h:427]: I think it is more reasonable to change the type of `_imageWidth ` and `_imageHeight ` to float. 
@ricardoquesada what do you think?

File: cocos/2d/CCLabel.h (hunk 1 above)
Comment (ricardoquesada) [reply cocos/2d/CCLabel.h]: agreed with @minggo

File: cocos/3d/CCTerrain.h (hunk 2 above)
Comment (ricardoquesada) [reply cocos/3d/CCTerrain.h:427]: @minggo the code, internally, treat it as int. in this particular case, it seems better to cast it to float in `getTerrainSize`

Comment (minggo) [reply cocos/3d/CCTerrain.h:427]: Agree with @ricardoquesada

File: cocos/2d/CCLabel.h (hunk 1 above)
Comment (minggo) [reply cocos/2d/CCLabel.h]: @effolkronium what's your opinion? Could you please change to int to float?

Comment (ilqvya) [reply cocos/2d/CCLabel.h]: @minggo i've made the change. Float seems better.

**Pull request merged**